step5__run_haplotype_caller_per_sample.py

```diff
@@ -1,88 +1,142 @@
-import hail as hl   # used for hadoop file utils
 import logging
-import os
-import pandas as pd
-import tqdm
+from tqdm import tqdm
 
-from batch import batch_utils
+import hail as hl
 
-logging.basicConfig(format='%(asctime)s %(levelname)-8s %(message)s', level=logging.INFO)
-logger = logging.getLogger(__name__)
+from gnomad.resources.resource_utils import DataException
+from gnomad.utils.file_utils import call_parallel_file_exists
 
+from tgg.batch.batch_utils import (
+    check_storage_bucket_region,
+    HG38_REF_PATHS,
+    localize_file,
+    init_arg_parser,
+    init_job,
+    run_batch,
+    set_gcloud_project,
+)
 
-GCLOUD_PROJECT = "broad-mpg-gnomad"
-GCLOUD_USER_ACCOUNT = "weisburd@broadinstitute.org"
-GCLOUD_CREDENTIALS_LOCATION = "gs://weisburd-misc/creds"
 
-DOCKER_IMAGE = "gcr.io/broad-mpg-gnomad/gnomad-readviz@sha256:7013fc57e3471617a314b08e2bcefe4711d401f83500c5c57e9a3e79ee8efebd"
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("run_haplotypecaller")
+logger.setLevel(logging.INFO)
 
-#EXCLUDE_INTERVALS = "gs://gnomad-bw2/exclude_intervals_with_non_ACGT_bases_in_GRCh38.bed"
-EXCLUDE_INTERVALS = "gs://gnomad-bw2/exclude_intervals_with_non_ACGT_bases_in_GRCh38__150bp_window.bed"
+
+EXCLUDE_INTERVALS = (
+    "gs://gnomad-bw2/exclude_intervals_with_non_ACGT_bases_in_GRCh38__150bp_window.bed"
+)
+"""
```

Comment:
Finding it a little counterintuitive to see docs below the var definition. Is this the preferred style?

Reply:
I can't find the python article about this, but this is the format we use in the gnomad_methods repo: 
![image](https://user-images.githubusercontent.com/19142091/128230851-af25bb1a-3cba-4a9a-b9c9-68205cea9fa5.png)
(from https://github.com/broadinstitute/gnomad_methods/blob/master/gnomad/utils/vcf.py)

Reply:
got it